tethysext/atcore/models/app_users/spatial_resource.py

```diff
@@ -78,3 +78,11 @@ def get_extent(self, extent_type: str = 'dict'):
         if extent_type == 'dict':
             extent = json.loads(extent)
         return extent
+
+    def update_extent_srid(self, srid):
+        session = inspect(self).session
+        object_to_convert = session.query(Resource).filter(Resource.id == self.id).first()
```

Comment:
I'm pretty sure `object_to_convert` is `self`, no? If you need a new copy, use the `copy` module.

Reply:
In other words I don't think lines 83 and 84 are necessary. Just use `self`

Reply:
object_to_convert is resource. We need to do the query to get the geom extent

Reply:
got it.

Reply:
Actually, that should work. We are working on it now.